api/tests/test_email_address_scalar.py

```diff
@@ -0,0 +1,67 @@
+import sys
+import os
+
+import pytest
+from graphene.test import Client
+from graphql import GraphQLScalarType
+from graphql.language import ast
+from graphql import GraphQLError
+
+import unittest
+
+# This is the only way I could get imports to work for unit testing.  TODO: See if there is a better way!
+PACKAGE_PARENT = '..'
+SCRIPT_DIR = os.path.dirname(os.path.realpath(os.path.join(os.getcwd(), os.path.expanduser(__file__))))
+sys.path.append(os.path.normpath(os.path.join(SCRIPT_DIR, PACKAGE_PARENT)))
+
+from scalars.email_address import *
+
+
+class TestEmailAddressScalar(unittest.TestCase):
+
+	def testValidEmailSerialize(self):
```

Comment:
I believe that functions should follow snake_case conventions rather than camelCase.